core/nginx/dovecot/proxy.conf

```diff
@@ -161,4 +176,11 @@ service submission-login {
    inet_listener submission-webmail {
     port = 10025
    }
+  service_count = 0
+  client_limit = 25000
+  process_min_avail = {{ CPU_COUNT }}
+  process_limit = {{ CPU_COUNT }}
+  vsz_limit = 256M
 }
+
+!include_try /overrides/dovecot.conf
```

Comment:
https://github.com/Mailu/Mailu/blob/341afeb57d266c1d990d1c41a6c57f0d2852117a/docs/faq.rst?plain=1#L245
Documentation (see above link) should be extended that you can drop dovecot.conf in overrides/nginx to override dovecot in front.

Reply:
I'm unsure how it interacts with the nginx conf where we say that all '*.conf' will be imported into nginx

Should we exclude it somehow or let it be and accept the warning messages that may result?

Reply:
Maybe we can say it must be in an extra folder?
../overrides/nginx/dovecot/dovecot.conf
And then change the inc
`!include_try /overrides/dovecot/dovecot.conf`

Reply:
That's 12e94b76